src/parser/chiquito.lalrpop

```diff
@@ -37,9 +37,9 @@ pub Statements: Vec<Statement<BigInt, Identifier>> = {
 }
 
 ParseStatement: Statement<BigInt, Identifier> = {
-    <l: @L> <s: StatementType> ";" <r: @R> => add_dsym(dsym_factory.create(l,r), s),
```

Comment:
Each statement type already has its own L and R parsing, so `add_dsym` wasn't needed anymore.